src/components/product-list/ProductPriceInfo.tsx

```diff
@@ -0,0 +1,63 @@
+import { Product } from '@common/type';
+import { Box, Typography } from '@mui/material';
+import {
+  cardPriceInfoContainer,
+  priceWithoutDiscount,
+  productPrice,
+  shipingDurationInfo,
+  shippingPriceInfo,
+} from '@styles/all-products/productListItem';
+import ProductCardButtons from './ProductCardButtons';
+
+interface ProductPriceInfoProps {
+  product: Product;
+  pcs: boolean | undefined;
+}
+
+export default function ProductPriceInfo({
+  product,
+  pcs,
+}: ProductPriceInfoProps) {
+  const price = () => {
+    if (pcs) {
+      const priceNumber =
+        product.discount && product.price.pcs
+          ? ((product.price.pcs * product.discount) / 100).toFixed(2)
+          : product.price.pcs;
+      return priceNumber;
+    } else {
+      const priceNumber =
+        product.discount && product.price.kg
+          ? ((product.price.kg * product.discount) / 100).toFixed(2)
+          : product.price.kg;
+      return priceNumber;
+    }
+  };
+
+  const shippingPrice = product.shippingPrice
+    ? 'Shipping Price: ' + product.shippingPrice + ' USD'
```

Comment:
better practice to use template literals `'Shipping Price: ${product.shippingPrice} USD`